tests/direct_beam_test.py

```diff
@@ -6,6 +6,93 @@
 from ess import polarization as pol
 
 
+def dummy_analyzer_spin() -> pol.CellSpinLog[pol.Analyzer]:
+    time = sc.array(dims=['time'], values=[0, 500], unit='s')
+    spin = sc.array(dims=['time'], values=[-1, 1], unit=None)
+    return pol.CellSpinLog[pol.Analyzer](sc.DataArray(spin, coords={'time': time}))
+
+
+def dummy_polarizer_spin() -> pol.CellSpinLog[pol.Polarizer]:
+    time = sc.array(dims=['time'], values=[0, 250, 750], unit='s')
+    spin = sc.array(dims=['time'], values=[-1, 1, -1], unit=None)
+    return pol.CellSpinLog[pol.Polarizer](sc.DataArray(spin, coords={'time': time}))
+
+
+# Setup logs for for sections of length 250:
+# - 10 s direct beam no cell
+# - 20 s direct beam with polarizer
+# - 20 s direct beam with analyzer
+# - 200 s sample run
+section_length = sc.scalar(250.0, unit='s')
+
+
+def dummy_sample_in_beam() -> pol.SampleInBeamLog:
+    time = sc.array(
+        dims=['time'], values=[0, 50, 250, 300, 500, 550, 750, 800], unit='s'
```

Comment:
This is currently bin centers, should it be a bin-edge coord?

Reply:
I thought so at first, but I think we will typically get data from NeXus files that is not a histogram. This is why we interpolate like this:
```python
interp1d(log, 'time', kind='previous', fill_value='extrapolate')
```